### PR TITLE
ci: DLT-1751 move release to GitHub actions

### PR DESCRIPTION
**Pull request by juliodialpad**
# Move release to GitHub actions

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] CI

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1751

## :book: Description

- Refactored the package release process to be `scheduled` on a weekly basis but also enable to run `manually`.
- Updated  `tj-actions/branch-names` version.
- Removed the local release scripts. 

## :bulb: Context

We were having some issues releasing packages as sometimes the local environment might be different for any of us, this will make sure that the environment is clean on releasing and will get rid of such issues.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :crystal_ball: Next Steps

Do releases and test that everything is working properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -323,52 +323,53 @@ nx build dialtone-documentation
...
+#### Production
 
-If something goes wrong, and you need to re-run the release, you can run the workflow manually through the `Run workflow`
-option on GitHub.
+##### Scheduled
 
-Select the `production` branch and pass the `commit SHA` which you would like to be the base to detect the changes or
-choose a `package` to release individually.
+On every Tuesday at 10:00 am UTC, [release action](.github/workflows/release.yml) will trigger the production release process which
+automatically release all packages that need to be released following the next steps:
```
Comment (iropolo) [inline README.md:335]: How we can stop or deactivate this automatic release if we dont want to release this week?

**Review by iropolo — APPROVED** [review]

**Comment by iropolo** [comment]
Is there a way to test this GHA before send it to production?, maybe we can use another branch like alpha/beta/next?,
just in case to prevention, but should be ok, not 100% necessary...

File: README.md (hunk 1 above)
Comment (juliodialpad) [reply README.md:335]: It's possible to disable the workflow manually, we can do that if we don't want to release that week.

Screenshot 2024-05-14 at 9 50 16 a m

**Pull request merged**